File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.c
// Last updated: 1/1/2026, 10:53:34 PM
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize) {
    int wealthiest = 0;
    int wealth = 0;

    for (int i = 0; i < accountsSize; i++) {
        for (int j = 0; j < accountsColSize[i]; j++) {
            wealth += accounts[i][j];
        }
        if (wealth >= wealthiest) {
                wealthiest = wealth;
                wealth = 0;
        }
        else {
            wealth = 0;
        }
    }
    return wealthiest;
}

File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.py
# Last updated: 1/1/2026, 10:53:38 PM
class Solution(object):
    def maximumWealth(self, accounts):
        wealthiest = 0
        for i in accounts:
            wealth = 0
            for j in i:
                wealth += j

            if wealth >= wealthiest:
                wealthiest = wealth
        return wealthiest

        